scripts/Utility.js

```diff
@@ -0,0 +1,56 @@
+// Globally accessable helper functions
+
+'use strict';
+
+// format text for the badge and for the song URL
+function formatHour(time) {
+	if (time === -1) {
+		return '';
+	}
+	if (time === 0) {
+		return '12a';
+	}
+	if (time === 12) {
+		return '12p';
+	}
+	if (time < 13) {
+		return time + 'a';
+	}
+	return (time - 12) + 'p';
+}
+
+// get current weather conditions using openweathermap: http://openweathermap.org/current
+function updateWeatherCond(zip, country, cb) {
```

Comment:
I don't think this belongs as a Utility function. My vision for this file was to be functions that are useful to multiple parts of the program, and thus belong in the global namespace. This and getSrc should only be seen by individual parts of the program, and should probably be encapsulated in the objects that use them.